docs/zh/deploy/install_deploy.md

```diff
@@ -83,6 +83,16 @@ Wed Aug 22 16:33:50 CST 2018
 ```
 请确保时间是正确的
 
+## 网络白名单
+
+OpenMLDB集群的服务组件之间需要保证网络连通。
+
+客户端到OpenMLDB集群则分为两种情况：
+
+- client（CLI/各SDK）连接OpenMLDB集群，除了与zookeeper的连通，还需要保证和nameserver/tabletserver/taskmanager的连通，因为client操作可能会直连某个server。
```

Comment:
let's use the formal and consistent names: ZooKeeper, TaskManager, APIServer, IP, etc.